src/components/pages/pitch-arena/PitchNavigation.tsx

```
@@ -0,0 +1,236 @@
+                  </div>
+
+                  <div className="flex-1 overflow-y-auto px-4 py-6 space-y-2">
+                    {navItems.map((n) => (
+                      <button
+                        key={n.id}
+                        onClick={() => {
+                          onNavigate(n.id);
+                          setOpen(false);
+                        }}
+                        className="w-full text-left rounded-lg px-4 py-3 hover:bg-muted/50 transition flex items-center gap-3"
+                      >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)